how2meet/db/schemas.py

```diff
@@ -31,7 +31,7 @@ class EventCreate(Event):
     pass
 
 
-class EventUpdate(BaseModel):
+class EventUpdate(Event):
```

Comment:
I'm a little confused by the consequences of overwriting all of the original class fields with these pydantic `Field` instances. I guess if you tested it and it didn't error then it's not a blocker for now.